src/redux/configureStore.js

```diff
@@ -0,0 +1,16 @@
+import { createStore, combineReducers, applyMiddleware } from 'redux';
```

Comment:
- [ ] From the project requirement you should Import the necessary methods from Redux Toolkit like Imports `configureStore `from `@reduxjs/toolkit`.  Please fix it :smile: 
![Screenshot from 2022-11-14 22-12-04](https://user-images.githubusercontent.com/6615394/201779109-88f9c63e-cda8-4456-bb91-7526c99449de.png)